Quaver.Shared/Skinning/SkinKeys.cs

```diff
@@ -543,8 +547,16 @@ private void LoadLaneSpecificElements()
                     ColumnColors[i] = ConfigHelper.ReadColor(ColumnColors[i], Store.Config[ModeHelper.ToShortHand(Mode).ToUpper()][$"ColumnColor{i + 1}"]);
 
                 // HitObjects
-                LoadHitObjects(NoteHitObjects, $"note-hitobject-{i + 1}", i);
-                LoadHitObjects(NoteHoldHitObjects, $"note-holdhitobject-{i + 1}", i);
+                if (UseSameHitObjects == true)
```

Comment:
This doesn't look right. I think if this flag is enabled, then the Hold objects should use the normal ones - currently it's the other way around.

Reply:
Fixed that